library/vulnerabilities/attack-wave-detection/checkQuery.ts

```diff
@@ -0,0 +1,45 @@
+import type { Context } from "../../agent/Context";
+import { extractStringsFromUserInputCached } from "../../helpers/extractStringsFromUserInputCached";
+
+const keywords = [
+  "SELECT (CASE WHEN",
+  "SELECT COUNT(",
+  "SLEEP(",
+  "WAITFOR DELAY",
+  "SELECT LIKE(CHAR(",
+  "INFORMATION_SCHEMA.COLUMNS",
+  "INFORMATION_SCHEMA.TABLES",
+  "MD5(",
+  "DBMS_PIPE.RECEIVE_MESSAGE",
+  "SYSIBM.SYSTABLES",
+  "RANDOMBLOB(",
+  "SELECT * FROM",
+  "1'='1",
+  "PG_SLEEP(",
+  "UNION ALL SELECT",
+  "../",
```

Comment:
should we keep this for `isWebScanPath`? do the scanners try `../` inside query parameters?